first draft of readme

Jneats opened the pull request.

jryusuf commented on README.md:
```diff
@@ -1,42 +1,186 @@
+### Prerequisites
 
-As far as possible, the project should be deployed automatically using infrastucture-as-code and CI/CD techniques.  
+* Python - [Installation guide](https://www.python.org/downloads/)
```
We can make installation framework independent or may be include other installation ways as well like using pip, brew is not installed by default in linux or windows and requires additional step

Jneats commented:
Added diagram. Added text to explain when to use brew to install python vs following the installation guide. Once merged into main, will be able to get image URL to add to the readme.
